From 615816942190923d911edf397334bdc170a1178d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Mon, 10 Jan 2011 16:38:22 +0100
Subject: [PATCH] Document --always-resolve at edquota and setquota

This copy --always-resolve option description for quota(1) into
edquota(8) and setquota(8) manuals.
---
 edquota.8  |    4 ++++
 setquota.8 |    4 ++++
 2 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/edquota.8 b/edquota.8
index d31d994..9b7ffd4 100644
--- a/edquota.8
+++ b/edquota.8
@@ -116,6 +116,10 @@ Duplicate the quotas of the prototypical user
 specified for each user specified.  This is the normal
 mechanism used to initialize quotas for groups of users.
 .TP
+.B --always-resolve
+Always try to translate user / group name to uid / gid even if the name
+is composed of digits only.
+.TP
 .B -F, --format=\f2format-name\f1
 Edit quota for specified format (ie. don't perform format autodetection).
 Possible format names are:
diff --git a/setquota.8 b/setquota.8
index 624deec..c7fe408 100644
--- a/setquota.8
+++ b/setquota.8
@@ -155,6 +155,10 @@ Use quota settings of user or group
 .I protoname
 to set the quota for the named user or group.
 .TP
+.B --always-resolve
+Always try to translate user / group name to uid / gid even if the name
+is composed of digits only.
+.TP
 .B -b, --batch
 Read information to set from stdin (input format is
 .I name block-softlimit block-hardlimit inode-softlimit inode-hardlimit
-- 
1.7.3.4

